stat/sampleuv/example_burnin_test.go

```diff
@@ -29,9 +32,9 @@ func ExampleMetropolisHastings_burnin() {
 	// typical problems, if Sigma is too small, it takes a lot of samples
 	// to move around the distribution. If Sigma is too large, it can be hard
 	// to find acceptable samples.
-	proposal := ProposalDist{Sigma: 0.2}
+	proposal := sampleuv.ProposalDist{Sigma: 0.2}
```

Comment:
This is actually defined in this file. It's also used in the rate example which presumably also needs to be made a sampleuv_test package member.